src/display/api.js

```diff
@@ -1148,6 +1148,11 @@ var PDFWorker = (function PDFWorkerClosure() {
     return PDFJS.fakeWorkerFilesLoadedCapability.promise;
   }
 
+  function createCDNWrapper(url) {
+    var wrapper = 'importScripts(\'' + url + '\');';
+    return URL.createObjectURL(new Blob([wrapper]));
```

Comment:
Not really, I want it to fail here since `blob:` scheme specifies its origin -- our polyfill does not.